src/services/firebase-client/rtdb/index.ts

```diff
@@ -0,0 +1,49 @@
+import firebase from "firebase/app"
+import "firebase/database"
+import { configs as firebaseConfigs } from '../../../constants/firebase'
+
+const isFirebaseInitialized = (): boolean => !!firebase.apps.length
+
+if (!isFirebaseInitialized()) {
+  firebase.initializeApp(firebaseConfigs)
+}
+
+const db = firebase.database()
+
+
+export const create = async (key: string, value: any) => {
+  const error = await db.ref(key).set(value)
+
+  if (error) {
+    console.error('Data could not be saved')
+    console.error(error)
+  }
+}
+
+export const append = (key: string, value: any) => {
+  db.ref(key).push(value)
+}
+
+
+export const get = async (key: string) => {
+  let result = null
+
+  try {
+    const snapshot = await db.ref(key).get()
+    result = snapshot.exists() ? snapshot.val() : null
+  } catch (error) {
+    console.error('Data could not be retrieved')
+    console.error(error)
```

Comment:
unfamiliar with js handling
should there be an error handler from argument to trigger error handler?

Reply:
i could do that, but for now i am going to keep it simple by simply console.log the error, and not specifically handle them

Reply:
Then, i dont think this file is necessary